common/app/views/mainAMP.scala.html

```diff
@@ -40,6 +42,22 @@
 
             @body
 
+            <div class="fc-container__inner">
+                <div class="fc-container__header__title">
+                    <span>popular</span>
+                </div>
+                <amp-list width="600" height="1000" layout="responsive" src="@if(Play.isDev) {https://www.thegulocal.com} else {@Configuration.ajax.url}/most-read-mf2.json">
```

Comment:
That makes sense! As you did previously, I will raise the issue with AMP to allow a relative path here, as long as the currently path is https.